README.md

```diff
@@ -153,7 +154,24 @@ def handler(event, context):
   context.iopipe.label('this-invocation-is-special')
 ```
 
-## Plugins
+### Core Agent
+
+By default, the IOpipe agent comes pre-loaded with all the bundled plugins in `iopipe.contrib.*`. If you prefer to run the agent without plugins or configure which plugins are used, you can use `IOpipeCore`:
```

Comment:
Shouldn't this be choosy about what it includes, if it's following the model of the node agent?

Reply:
Addendum: should we mention the plugins it includes?

Reply:
The docs say the preloaded plugins are the ones in `iopipe.contrib.*`, but I can be more explicit in the docs. We don't need to exclude the profiler plugin from the Python agent as it's disabled by default and adds no overhead when preloaded.